### ISSUE TITLE
Error: page.reload: Target page, context or browser has been closed

### ISSUE THREAD

**Issue by royaltongue** [TheNetsky/Microsoft-Rewards-Script#66]
```
[2/15/2024, 3:48:20 PM] [PID: 149935] [LOG] [SEARCH-BING] 5 Points Remaining | Query: love is blind season 6 | Mobile: true
[2/15/2024, 3:48:55 PM] [PID: 149935] [LOG] [SEARCH-BING] Completed searches
[2/15/2024, 3:48:58 PM] [PID: 149935] [LOG] [MAIN-POINTS] The script collected 50 points today
[2/15/2024, 3:48:59 PM] [PID: 149935] [ERROR] [GET-EARNABLE-POINTS] An error occurred:Error: page.reload: Target page, context or browser has been closed
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "undefined".] {
  code: 'ERR_UNHANDLED_REJECTION'
}

Node.js v21.6.1
```

The script says its finished its searches, then crashes with the error above. However, upon checking my account, my PC searches aren't complete (currently sitting at 60/150 points)

EDIT: Just realized these were mobile searches, which are 100% complete on my account for the day. Still, PC searches are not

**Comment by basalisco** [comment]
```
[21/2/2024, 16:52:10] [PID: 5620] [ERROR] [GET-EARNABLE-POINTS] An error occurred:TypeError: Cannot read properties of undefined (reading 'forEach')
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "undefined".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
```

same problem but only with a specific account

**Comment by TheNetsky** [comment]
Fixed 1.4.5
